### ISSUE TITLE
[Add Media] Effect polymorphism with Arrow FX

### ISSUE THREAD

**Issue by colomboe** [arrow-kt/arrow#1517]
**Type of Content:** blog post

**Link:** https://www.msec.it/blog/effect-polymorphism-with-arrow-fx/

**Brief Description of Content:** Wonder how to handle side effects in a very clean way while abstracting the real effect implementation? #Arrow #FX provides an easy way to do this, without the burden of Higher-Kinded Types.

**Author/Presenter Name:** Emanuele Colombo

**Twitter handle:** https://twitter.com/colomboe/status/1151815338708930561

**Comment by pakoito** [comment]
Wait, it should be easy to PR it to the blog. You did it for the previous post, right?

**Comment by colomboe** [comment]
> Wait, it should be easy to PR it to the blog. You did it for the previous post, right?

Yes, I thought this was the new "procedure"... Do you prefer a PR?

**Comment by pakoito** [comment]
Yes, if you don't mind!

**Comment by colomboe** [comment]
Ok, no problem, but I'll proceed on monday
